dvc/command/plots.py

```diff
@@ -98,7 +98,7 @@ def run(self):
 
 def add_parser(subparsers, parent_parser):
     PLOTS_HELP = (
-        "Generating plots for metrics stored in structured files "
+        "Commands to visualize and compare plot metrics in structured files "
```

Comment:
Thanks for this change. We may be changing it further per this https://github.com/iterative/dvc.org/pull/1382#discussion_r434966620. Please check the resolution tomorrow or so.